proto/beacon/rpc/v1/services.proto

```diff
@@ -16,6 +16,10 @@ service ProposerService {
   rpc ProposeBlock(ethereum.eth.v1alpha1.BeaconBlock) returns (ProposeResponse);
 }
 
+service AggregatorService {
```

Comment:
Why create a new service? Shouldn't this be part of the validator service?

Reply:
I think Aggregator should have its own role (ie. similar to proposer and attester). It makes it more extensible as the aggregating spec gets more complicated and we adopt real-life aggregation protocol. Another reason is aggregator perform time based role (ie. submits signed slot signature when it's 2/3 of the slot)

Reply:
Thanks!